### DISCUSSION TITLE
2D Camera information

### DISCUSSION THREAD

**Discussion by Xindictus** [gstavrinos/ai_msc_robotic_systems_assignments#32] [Q&A] [answered]
1. I searched for the camera's `URDF` in the workspace, but, I couldn't find it. Does it reside somewhere else?
2. Do we need the focal length of the camera?
3. Is it possible to get the depth of an object using just the `2D` camera? If yes, do we need to calibrate the camera for that?

**Comment by gstavrinos** [comment, marked as answer]
1. The camera's URDF is located in the robot description's package. It is not mandatory to inspect it, in order to find a solution though.
2. The focal length of the camera is not required but could lead to a solution (URDF could help you with that, and most specifically the simulated parameters of the camera). I would not try to handle it this way though.
3. It is possible to get the depth using a 2D camera, but apart from the need of calibration you would also not get the best results, given the fact that your robot already has a 3D sensor. Maybe try combining the 2D camera detection with is 3D sibling.
